### PR TITLE
TAXI-16 Model trip_distance

### PR DESCRIPTION
**Pull request by dk1000**

### CODE REVIEW

File: src/features/trip_distance.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        self.model_dir = model_dir
+        self.model_path = get_model_path() / self.model_dir
+        self.model_name = None
+        self.model = None
+
+        self.model_name = f"trip_distance_model.pkl"
+        self.model_columns = model_columns
+        self.max_depth = model_max_depth
+
+    def fit(self, X: pd.DataFrame, y: pd.Series = None):
```
Comment (dk1000) [inline src/features/trip_distance.py:33]: that part I don't like, do you have idea how to better pass X and y? Now I take y from X and then remove redundant columns in 37/38

**Comment by AdamJedz** [comment]
Why haven't you used pickup date columns? Distance may vary on whether there are rush hours or not.
TBH I don't really like simplifying this model. I understand sticking with only RF, but we should have there some hyperparameters optimization based on cross-validation (it does not have to be complicated)

File: src/features/trip_distance.py (hunk 1 above)
Comment (AdamJedz) [reply src/features/trip_distance.py:33]: i think that since we are using this model as a feature transformer, this is the proper way

**Review by AdamJedz — APPROVED** [review]
ok

**Pull request merged**